doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -229,9 +229,9 @@ To program the Secure Domain Firmware (SDFW, also known as ``urot``) and the Sys
    .. note::
       On MacOS, ensure that the ZIP file is not unpacked automatically upon download.
 
-#. Move the :file:`.zip` bundle to a folder of your choice, then run nRF Util to program the binaries using the following command::
+#. Move the bundle's :file:`.zip` archive to a folder of your choice, unzip the archive, and then run nRF Util to program the :file:`nordic_top.suit` using the following command::
 
-      nrfutil device x-provision-nrf54h --firmware <path-to_bundle_zip_file> --serial-number <serial_number>
+        nrfutil device x-suit-dfu --firmware nordic_top.suit --serial-number <serialnumber>
```

Comment:
The old description was correct (run x-provision-nrf54h with the zip file), the new one is incorrect.

Reply:
@Tiwalun @kemegard Can you clarify this?

Reply:
@bjorgan is corrct, if you want to install the secure domain FW on a fresh device, you need to use the `x-provision-nrf54h` command, with the zip file.